.github/workflows/deploy.yaml

```diff
@@ -2,18 +2,32 @@ name: Deployment
 
 on:
   workflow_dispatch:
+  pull_request:
+    paths:
+      - "lib/**"
+      - "integration_tests/**"
```

Comment:
why would we run the deployment workflow if we just update the integration test?

Reply:
ah nevermind, I forgot the integration tests are where we actually run the deployment :facepalm: